streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -108,7 +109,7 @@
 
     private WindowStore<Integer, String> createWindowStore(final ProcessorContext context, final boolean retainDuplicates) {
         final WindowStore<Integer, String> store = Stores.windowStoreBuilder(
-            Stores.persistentWindowStore(windowName, retentionPeriod, windowSize, retainDuplicates, segmentInterval),
```

Comment:
@dguy Yes, this is where I've updated it.

Reply:
I think that method is private?? I might have missed somerhing

Reply:
It was previously public. #5804 made it private, but did not remove this reference. This is why trunk fails to build right now.

I'm removing this reference and instead calling a public method.